src/rgw/rgw_sal_sfs.h

```diff
@@ -363,17 +360,12 @@ class SFStore : public Store {
       info.bucket.marker = info.bucket.bucket_id = 
         bucket.name + "." + std::to_string(ts.tv_sec) + std::to_string(ts.tv_nsec);
 
-    info.owner = owner.user_id;
+    info.owner = db_binfo.binfo.owner;
     info.zonegroup = db_binfo.binfo.zonegroup;
     info.creation_time = db_binfo.binfo.creation_time;
-    info.placement_rule = placement_rule;
```

Comment:
This might have been mistakenly removed. I don't think we want to drop this line.

Reply:
ugh, nice catch.

Reply:
this would have been prevented if python tests would have run on CI on push; this is something we will have to add to CI :smirk:

Reply:
@jecluis checked better this, and that remove is intended.
now that code has been substituted with:

`info.placement_rule = db_binfo.binfo.placement_rule;`

I have run the bucket python tests and they must be fixed in agreement with the removal of version attributes.
But this is another story.